From df1ed7db4c56e601d1f56405a0c7def98a01693f Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 11 May 2019 10:27:58 +0200
Subject: [PATCH 1/1] efi_loader: reword the EFI_LOADER config option

No need to mention U-Boot in brief description.
Fix several typos, mention iPXE.

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 lib/efi_loader/Kconfig | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/lib/efi_loader/Kconfig b/lib/efi_loader/Kconfig
index 03b082750f..1092bd2061 100644
--- a/lib/efi_loader/Kconfig
+++ b/lib/efi_loader/Kconfig
@@ -1,5 +1,5 @@
 config EFI_LOADER
-	bool "Support running EFI Applications in U-Boot"
+	bool "Support running UEFI applications"
 	depends on (ARM || X86 || RISCV || SANDBOX) && OF_LIBFDT
 	# We need EFI_STUB_64BIT to be set on x86_64 with EFI_STUB
 	depends on !EFI_STUB || !X86_64 || EFI_STUB_64BIT
@@ -11,10 +11,10 @@ config EFI_LOADER
 	select REGEX
 	imply CFB_CONSOLE_ANSI
 	help
-	  Select this option if you want to run EFI applications (like grub2)
-	  on top of U-Boot. If this option is enabled, U-Boot will expose EFI
-	  interfaces to a loaded EFI application, enabling it to reuse U-Boot's
-	  device drivers.
+	  Select this option if you want to run UEFI applications (like GNU
+	  GRUB or iPXE) on top of U-Boot. If this option is enabled, U-Boot
+	  will expose the UEFI API to a loaded application, enabling it to
+	  reuse U-Boot's device drivers.
 
 if EFI_LOADER
 
-- 
2.20.1

